index.yaml

```diff
@@ -1158,12 +1503,12 @@ entries:
       catalog.rancher.io/namespace: cattle-istio-system
       catalog.rancher.io/release-name: rancher-kiali-server
     apiVersion: v2
-    appVersion: v1.29.0
-    created: "2021-03-03T21:31:14.807153-07:00"
```

Comment:
These deletions are odd, but it looks like its just a weird way git did re-ordering. The original digest is still there. Can you confirm.

Reply:
v1.29.0 (https://github.com/rancher/charts/pull/1164/files#diff-08c65c1b5ef8384ef0200558777c52f4110afcbbacc880ac5375d555ca97ee3eR1541) and v.1.24.0 (https://github.com/rancher/charts/pull/1164/files#diff-08c65c1b5ef8384ef0200558777c52f4110afcbbacc880ac5375d555ca97ee3eR1576) are below